project/UI/Carbon/Carbon.swift

```diff
@@ -22,23 +22,23 @@ public struct CarbonCommand: ParsableCommand {
     @ArgumentParser.Option(help: "Path where the resulting carbon files will be generated")
     private var output: ArgumentPath
     
-    @ArgumentParser.Option(default: "nef", help: "Background color in hexadecimal")
-    private var background: String
+    @ArgumentParser.Option(help: "Background color in hexadecimal")
+    private var background: String = "nef"
     
-    @ArgumentParser.Option(default: .dracula, help: "Carbon theme")
-    private var theme: CarbonStyle.Theme
+    @ArgumentParser.Option(help: "Carbon theme")
+    private var theme: CarbonStyle.Theme = .dracula
     
-    @ArgumentParser.Option(default: .x2, help: "export file size [1-5]")
-    private var size: CarbonStyle.Size
+    @ArgumentParser.Option(help: "export file size [1-5]")
+    private var size: CarbonStyle.Size = .x2
     
-    @ArgumentParser.Option(default: .firaCode, help: "Carbon font type")
-    private var font: CarbonStyle.Font
+    @ArgumentParser.Option(help: "Carbon font type")
+    private var font: CarbonStyle.Font = .firaCode
     
-    @ArgumentParser.Option(name: .customLong("show-lines"), default: true, help: "Shows/hides lines of code [true | false]")
-    private var lines: Bool
+    @ArgumentParser.Option(name: .customLong("show-lines"), help: "Shows/hides lines of code [true | false]")
```

Comment:
This should be a flag with the corresponding inversion property.

Reply:
mmmm it is weird, I thought it has done! I will review it

Reply:
Yep, it was done in a reviewed [PR](https://github.com/bow-swift/nef/pull/167#discussion_r463543065) about command-line autocomplete. It was a previous one - you can see the pending-to-merge branch `shell_autocomplete+spm_nefc`